astropy/io/fits/header.py

```diff
@@ -1359,7 +1359,9 @@ def extend(self, cards, strip=True, unique=False, update=False,
                 else:
                     extend_cards.append(card)
             else:
-                if unique or update and keyword in self:
+                # First parentheses needed because of operator precedence.
+                # Second is just for clarity.
+                if (unique or update) and keyword in self:
```

Comment:
But I only see one pair of parens :)  
I think the comment could be removed--the intent of the code was clear I think, I just forgot to put them in in the first place (but if you do think it merits a comment I think the comment should be reworded because I'm confused by the mentiono f a "second" parentheses :)